public/apps/configuration/utils/index-permission-utils.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import { map } from 'lodash';
+import { FieldLevelSecurityMethod, TenantPermissionType } from '../panels/role-edit/types';
```

Comment:
Suggest we also move these 2 type up, generally we should not import types from lower level.

Reply:
moved to upper level and refactored code.